getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
```

Comment:
> While these can appear in various forms, from log lines to complex traces

Again, I feel the same way as above. This innocuous "from" scopes the breadth of use cases to engineering when I read this. The framing is changed from what I want the world to know, which is that every function in any business with digital activity can and will benefit from Axiom.